indico/modules/events/abstracts/templates/management/overview.html

```diff
@@ -1,118 +1,155 @@
-            </div>
-            <div class="toolbar">
-                <div class="group">
-                    <a href="#" class="i-button icon-mail"
-                       data-href="{{ url_for('.email_tpl_list', event) }}"
-                       data-title="{% trans %}Configure e-mail notifications{% endtrans %}"
-                       data-ajax-dialog>
-                        {% trans %}Notifications{% endtrans %}
-                    </a>
+    {% if not event.has_feature('abstracts') %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-file-check"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Call for Abstracts disabled{% endtrans %}
+                    </div>
+                    {% trans %}"Call for Abstracts" provides an integrated abstract management and reviewing module that will allow you to set up a full workflow.{% endtrans %}
                 </div>
-                <div class="group">
-                    <a href="#" class="i-button icon-list js-dialog-action"
-                       data-title="{% trans %}Manage fields{% endtrans %}"
-                       data-href="{{ url_for('contributions.manage_fields', event) }}"
-                       data-ajax-dialog
-                       data-reload-after="customData">
-                        {% trans %}Abstract fields{% endtrans %}
-                    </a>
-                </div>
-                <div class="group">
-                    <a href="#" class="i-button icon-settings"
-                       data-href="{{ url_for('.manage_submission_settings', event) }}"
-                       data-title="{% trans %}Configure abstract submission{% endtrans %}"
-                       data-ajax-dialog>
-                        {% trans %}Settings{% endtrans %}
+                <div class="toolbar">
+                    <a href="#" id="enable-abstracts" class="i-button highlight"
+                       data-href="{{ url_for('event_features.switch', event, feature='abstracts')  }}"
+                       data-method="PUT">
+                        {% trans %}Enable module{% endtrans %}
                     </a>
                 </div>
             </div>
         </div>
+        <script>
+            $('#enable-abstracts').on('click', function(evt) {
+                evt.preventDefault();
 
-        <div class="section">
-            <div class="icon icon-file-check"></div>
-            <div class="text">
-                <div class="label">
-                    {% trans %}Reviewing{% endtrans %}
+                var $this = $(this);
+                $.ajax({
+                    url: $this.data('href'),
+                    method: $this.data('method'),
+                    complete: function() {
+                        IndicoUI.Dialogs.Util.progress(),
```

Comment:
This doesn't work as you expect it to. `progress()` returns the function to close the progress dialog. Reload in the `success` callback instead (and call `progress()` in there again before reloading).